client/js/gdpr/chart/index.js

```diff
@@ -163,7 +167,7 @@ function createSeries(color, history) {
     series.append(marker);
     series.classList.add('series-active');
     marker.setAttribute('cx', x * RUN_SCALE_X + 'px');
-    marker.setAttribute('cy', y * RUN_SCALE_Y + 'px');
+    marker.setAttribute('cy', (y - 0.5) * RUN_SCALE_Y + 'px');
```

Comment:
what's the 0.5 changes for? How are adding date lines shifting everything by 0.5?

Reply:
this centers all of the lines vertically in the chart, whereas before they were placed too far down

Reply:
nit: make a variable that conveys that info or leave it in a comment:
```
const gridAlignedY = (y - 0.5) * RUN_SCALE_Y;
marker.setAttribute('cy', gridAlignedY + 'px');
...
// Y axis should be dropped half a scaled unit to vertically align it with the grid.
marker.setAttribute('cy', (y - 0.5) * RUN_SCALE_Y + 'px');
```